docs/source/training_eval.md

```diff
@@ -47,8 +65,7 @@ recsys_trainer = Trainer(
 recsys_trainer.train()
 ```
 
-
-You can optionally get the data loaders instantiated by the `Trainer` when the following arguments are provided.
+You can instantiate the dataloader when you create the {class}`Trainer <transformers4rec.torch.trainer.Trainer>` instance by specifying the arguments shown in the following code block when you create an instance of the {class}`T4RecTrainingArguments <transformers4rec.config.trainer.T4RecTrainingArguments>` class:
```

Comment:
The main idea here is that the user can directly pass the path and/or dataset of train and eval data using the arguments `train_dataset_or_path` and `eval_dataset_or_path`. In that case, the creation of the data loaders will be automatically handled by the `Trainer` class. However, to use this option, the user must also pass the `schema` to the trainer. 
```
# Instantiates the train and eval dataloader
Trainer(
    model=model,
    schema=schema,
    args=training_args,
    train_dataset_or_path=train_path,
    eval_dataset_or_path=eval_path,
)
```

Reply:
Corrected.  Thank you for the clarification.